Libraries/Text/Text.js

```diff
@@ -317,6 +326,12 @@ const Text = React.createClass({
         isHighlighted: this.state.isHighlighted,
       };
     }
+    if (typeof newProps.selectionColor !== 'undefined') {
```

Comment:
Ahh I see, Could you just change the if check to be newProps.selectionColor != null instead to match null and undefined.